app/src/main/java/org/simple/clinic/ClinicApp.kt

```diff
@@ -51,6 +55,12 @@ abstract class ClinicApp : Application(), CameraXConfig.Provider {
 
     crashReporter.init(this)
     Timber.plant(CrashBreadcrumbsTimberTree(crashReporter))
+    RxJavaPlugins.setErrorHandler { error ->
+      if (!error.canBeIgnoredSafely()) {
+        val cause = if(error is UndeliverableException) error.cause else error
+        Thread.currentThread().uncaughtExceptionHandler?.uncaughtException(Thread.currentThread(), cause!!)
```

Comment:
Can you tell me why we are doing this here, please?

Reply:
Some errors we can safely ignore, others we have to manually throw and ensure that it gets reported. This is what is happening here.